proposed/dotnet-bootstrapper/dotnet-bootstrapper-path.md

```diff
@@ -0,0 +1,130 @@
+
+> 📓 Technical Detail: VS Behavior with Local Installs:
+>
+> Finally, we should consider where else VS may be impacted. Some code within Visual Studio is hard-coded to use the administrative installs, such as the template engine. The template engine is a standalone component, and it is ok for this tool to run with the admin logic.
+
+### Multi-level-lookup
+
+`DOTNET_MULTILEVEL_LOOKUP` was designed to enable or disable the host/muxer/`dotnet.exe`'s ability to find SDKs or Runtimes in other locations. We could re-enable the muxer to look specifically at the directory of local installs that are placed by `bootstrapper`, as proposed to be `DOTNET_HOME` in the other specification, [`dotnet-bootstrapper-cli-experience.md`](./dotnet-bootstrapper-cli-experience.md).
+
+This would enable the lookup to work without modification to the `PATH`. We could add back this feature in a limited capacity and only with explicit user consent. However, `DOTNET_MULTILEVEL_LOOKUP` was deprecated and removed in .NET 7 due to user confusion (when it would find global SDKs in a user local context) and for performance reasons. The `host` team and others indicated that this feature presented significant challenges when it was available. When installers set conflicting paths, it creates coordination issues between different installation mechanisms, so it may be better to view this as an installer problem like above.
+
+### Require `paths` in `global.json`
+
+In this approach, we prioritize defined and replicable behavior. System administrator installs are created in predictable locations, especially those from Visual Studio and/or package manager feeds.
+
+What we could do is look for those installations and decline to do anything when a user tries to run `bootstrapper` if they are present on disk. The error message would point the user toward leveraging the `sdk: paths []` option in their project or repositories `global.json` file, which would specify to the host or `dotnet.exe` where to look for the installs of the .NET SDK. The host is already aware of this as of .NET 10.
+
+> 📓 Usage Statistic:
+> Only ~4% of users are on a `global.json` according to internal metrics, so this gesture will create additional work when using `dotnet` for a large number of customers.
+
+The downside of this approach is that it creates an additional step for a significant number of .NET users. Requiring extra configuration files to function properly presents a usability challenge and increases the barrier to entry for new users. However, the `paths` feature itself provides considerable value for repository-specific SDK management, which is why it has already been implemented and made available.
```

Comment:
Another downside is that as currently implemented, the `paths` feature only supports repo installs and not user installs (because it doesn't support variables/tokens to point to something like the home or LOCALAPPDATA fodler.  This means that different repos would not be able to share installations of the .NET SDK, wasting disk space for developers that use multiple repos.